pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -29,17 +32,30 @@
 from google.cloud.pubsub_v1.gapic import publisher_client
 from google.cloud.pubsub_v1.gapic.transports import publisher_grpc_transport
 from google.cloud.pubsub_v1.publisher._batch import thread
-
+from google.cloud.pubsub_v1.publisher._sequencer import ordered_sequencer
+from google.cloud.pubsub_v1.publisher._sequencer import unordered_sequencer
```

Comment:
**Idea:**
We could also import sequencer classes in `_sequencer/__init__.py`, and then refer to them as `_sequencer.OrderSequencer`, `_sequencer.UnorderedSequencer`, etc.,

This is less verbose than the current `ordered_sequencer.OrderedSequencer` and `unordered_sequencer.UnorderedSequencer`.

It's a minor thing, though, and by no means necessary.

Reply:
I like this current way for uniformity.